blocks/components/editable/dom-to-react.js

```diff
@@ -0,0 +1,41 @@
+import { createElement } from 'react';
+import { map, reduce } from 'lodash';
+import camelCaseAttrMap from 'html-to-react/lib/camel-case-attribute-names';
+
+export function nodeToReact( node, index ) {
+	if ( ! node ) {
+		return null;
+	}
+
+	if ( node.nodeName === '#text' ) {
+		return node.nodeValue;
+	}
+
+	if ( node.getAttribute( 'data-mce-bogus' ) === 'all' ) {
+		return null;
+	}
+
+	const type = node.nodeName.toLowerCase();
+	const children = map( node.childNodes || [], nodeToReact );
+	const props = reduce( node.attributes, ( result, { name, value } ) => {
+		let key = camelCaseAttrMap[ name.replace( /[-:]/, '' ) ] || name;
+
+		if ( key.startsWith( 'data-mce-' ) ) {
```

Comment:
We don't use `babel-polyfill`, so we don't have access to ES2015 base type prototype methods like `Array#includes` or `String#startsWith`. This will error in IE11. We may consider incorporating the polyfill. It's not small though. But this is a common enough problem that it's probably worth considering since it's easy for a developer to overlook. That or some custom lint behaviors.